xbmc/filesystem/CurlFile.cpp

```diff
@@ -593,6 +593,10 @@ void CCurlFile::SetCommonOptions(CReadState* state)
   if( m_contentencoding.length() > 0 )
     g_curlInterface.easy_setopt(h, CURLOPT_ENCODING, m_contentencoding.c_str());
 
+  // setup Accept-Encoding if requested
+  if (m_acceptencoding.length() > 0)
+    g_curlInterface.easy_setopt(h, CURLOPT_ACCEPT_ENCODING, m_contentencoding.c_str());
```

Comment:
I think you've a pasto here should be m_acceptencoding.c_str() (coverity pointed it out, but I suspect it's right :)